lib/src/presentation/view/popular_movies.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import '../../core/util/states.dart';
+import '../bloc/movie_list_bloc.dart';
+import '../widget/custom_scaffold.dart';
+import '../widget/stream.dart';
+
+class PopularMovies extends StatefulWidget {
+  const PopularMovies({
+    super.key,
+    required this.bloc,
+  });
+
+  final MoviesBloc bloc;
+
+  static const double paddingInListView = 15;
+
+  @override
+  State<PopularMovies> createState() => _PopularMoviesState();
+}
+
+class _PopularMoviesState extends State<PopularMovies> {
+  late Stream<DataState> data;
+
+  @override
+  void initState() {
+    super.initState();
+    data = widget.bloc.allMovies;
```

Comment:
Review if you really need the late initiialization, this could be affect in case of a bad initialization where you can have an exception. 
why you are saving the data?